docs/docs/themes/api-reference.md

```diff
@@ -56,39 +55,86 @@ module.exports = themeOptions => {
 }
 ```
 
-Then, in your theme's `gatsby-node.js` you can access them as the second argument to `createPages`:
+While using the usual object export (`module.exports = {}`) in your theme means that you can run the theme standalone as it's own site, When using a function in your theme to accept options you will need to run the theme as part of an example site. See how the [theme authoring starter](https://github.com/gatsbyjs/gatsby/tree/master/themes/gatsby-starter-theme-workspace) handles this using Yarn Workspaces.
+
+### Accessing Options elsewhere
+
+Note that because themes are also plugins you can also access the options in any of the lifecycle methods taht you're used to. For example, in your theme's `gatsby-node.js` you can access the options as the second argument to `createPages`:
 
 ```js:title=gatsby-node.js
 exports.createPages = async ({ graphql, actions }, themeOptions) => {
   console.log(themeOptions)
 }
 ```
 
-## Component Shadowing
+## Shadowing
+
+Since themes are usually deployed as NPM packages that other people use in their sites, we need a way to modify certain files, such as React components, without making changes to the source code of the theme. This is called Shadowing. Shadowing is a filesystem-based API that allows us to replace one file with another at build time. For example, if we had a theme with a `Header` component we could replace that `Header` with our own by creating a single file using Shadowing.
```

Comment:
```suggestion
Since themes are usually deployed as npm packages that other people use in their sites, we need a way to modify certain files, such as React components, without making changes to the source code of the theme. This is called _Shadowing_.

Shadowing is a filesystem-based API that allows us to replace one file with another at build time. For example, if we had a theme with a `Header` component we could replace that `Header` with our own by creating a new file in our site and placing it in the correct location for Shadowing to find it.
```